packages/kbn-plugin-helpers/tasks/build/create_build.test.js

```diff
@@ -88,9 +88,11 @@ describe('creating the build', () => {
     const sassPath = 'public/styles.scss';
     const cssPath = resolve(PLUGIN_BUILD_TARGET, 'public/styles.css');
 
-    // set skipInstallDependencies to true for these tests
-    beforeEach(() => (PLUGIN.styleSheetToCompile = sassPath));
-    // set it back to false after
+    beforeEach(() => {
+      PLUGIN.skipInstallDependencies = true;
+      PLUGIN.styleSheetToCompile = sassPath;
+    });
+
     afterEach(() => {
       PLUGIN.styleSheetToCompile = undefined;
```

Comment:
should this revert it too?

Reply:
This is adding `skipInstallDependencies` - `styleSheetToCompile` was already there.

Reply:
what about PLUGIN.skipInstallDependencies = undefined?

Reply:
Oh, I see - not sure it's needed but pushed a fix.